SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SpatialGDKEditorSchemaGenerator.cpp

```diff
@@ -296,33 +297,65 @@ void InitClassPathToSchemaMap()
 	}
 }
 
+bool TryLoadClassForSchemaGeneration(FString ClassPath)
+{
+	const FSoftObjectPath ItemToReference(ClassPath);
+
+	// First check if the object is already loaded into memory.
+	UObject* const ResolvedObject = ItemToReference.ResolveObject();
+	UClass*  const LoadedClass = ResolvedObject ? nullptr : Cast<UClass>(ItemToReference.TryLoad());
+
+	// Only store classes that weren't currently loaded into memory.
+	if (LoadedClass)
+	{
+		// Don't allow the Garbage Collector to delete these objects until we are done generating schema.
+		LoadedClass->AddToRoot();
+		AdditionalSchemaGeneratedClasses.Add(LoadedClass);
+	}
+
+	// Return true if the class exists.
+	return ResolvedObject || LoadedClass;
+}
+
+void LoadDefaultGameModes()
+{
+	TArray<FString> GameModesToLoad{ "GlobalDefaultGameMode", "GlobalDefaultServerGameMode" };
+
+	for (FString GameMode : GameModesToLoad)
+	{
+		// Get the GameMode from the DefaultEngine.ini.
```

Comment:
Does this work if it's defined elsewhere in the `___Engine.ini` stack?

Reply:
Verified, removed from DefaultEngine.ini and then added to WindowsEngine.ini. The DefaultGameMode was still picked up so [Platform]Engine.ini are correctly used.